src/backend/distributed/shared_library_init.c

```diff
@@ -1612,6 +1613,16 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL | GUC_NOT_IN_SAMPLE,
 		NULL, NULL, NULL);
 
+	DefineCustomStringVariable(
+		"citus.extension_owner",
```

Comment:
I think I'd prefer calling this `citus.superuser`. It doesn't necessarily need to be the extension owner.